### PR TITLE
Cambiar pregunta

### PR DESCRIPTION
**Pull request by matesuiso**

### CODE REVIEW

File: src/main/java/trivia/App.java [hunk 1]
```diff
@@ -230,64 +249,66 @@ else if (j == 1) {
...
 
-          pregunta.set("pregunta", request.queryParams("cambiar0")).saveIt();
-          pregunta.set("respuestaCorrecta", request.queryParams("cambiar1")).saveIt();
-          pregunta.set("wrong1", request.queryParams("cambiar2")).saveIt();
-          if (request.queryParams("cambiar3") != null) {
-            pregunta.set("wrong2", request.queryParams("cambiar3")).saveIt();
+         pregunta.set("pregunta", request.queryParams("cambiar0")).saveIt();
+         pregunta.set("respuestaCorrecta", request.queryParams("cambiar1")).saveIt();
+         pregunta.set("wrong1", request.queryParams("cambiar2")).saveIt();
+         if (request.queryParams("cambiar3") != null) {
```
Comment (santiago-m) [inline src/main/java/trivia/App.java]: The value of an input element will never be null, if the input is empty then the value is '' (empty string). So if the value is '' , the correct way should be "pregunta.set("answerX", null).saveIt()" or something like that. 
Otherway in the database, the answer appears as '' and not as null.

**Review by santiago-m — COMMENTED** [review]
I'd leave a comment in line 293 of App.java

File: src/main/java/trivia/App.java (hunk 1 above)
Comment (matesuiso) [reply src/main/java/trivia/App.java]: You wrong.. If the html form is empty, the request.queryParams returns null..

Comment (matesuiso) [reply src/main/java/trivia/App.java]: You wrong.. If the html form is empty, the request.queryParams returns null..

Comment (matesuiso) [reply src/main/java/trivia/App.java]: My bad.. I understand it right now

File: src/main/java/trivia/Question.java [hunk 2]
```diff
@@ -10,6 +10,7 @@
 	private int cantOpciones = 1;
 
 	static{
+      blankToNull("wrong2", "wrong3");
```
Comment (matesuiso) [inline src/main/java/trivia/Question.java:13]: with only that fixes the problem ;)  @santiago-m

**Review by santiago-m — APPROVED** [review]
I think it would be all. I changed the pull request to merge to develop and not master.-

**Pull request merged**